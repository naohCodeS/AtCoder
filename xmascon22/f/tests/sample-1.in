4
0 10 300 0
10 0 5 400
300 5 0 20
0 400 20 0
