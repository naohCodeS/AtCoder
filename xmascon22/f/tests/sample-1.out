1 735 120200
